dkg/runner.go

```diff
@@ -65,6 +65,12 @@ func (r *runner) ProcessMsg(msg *SignedMessage) (bool, error) {
 				return true, errors.Wrap(err, "protocol finished without blame or keygen result")
 			}
 
+			if !r.isResharing() {
+				if err := r.config.Storage.SaveKeyGenOutput(o.ProtocolOutput); err != nil {
+					return false, err
+				}
+			}
+
```

Comment:
What if it's resharing? We also need to store the output if we are in the new committee right?